homework-classes/RepoView.js

```diff
@@ -19,8 +19,27 @@
      * @param {Object} repo A repository object.
      */
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
-      console.log('RepoView', repo);
+      this.container.innerHTML = ' ';
+      const ul = createAndAppend('ul', this.container, {
+        class: 'list-container',
+      });
+      
+      function createDetailLine(ul, labelText, value) {
```

Comment:
Instead of creating an embedded function like this, it is a best practice when using classes to create a method. In this case, that can be a **static** method, as the function/method body does not reference the **this** value:

```js
render(repo) {
  const first = RepoView.createDetailLine(ul, 'Repository', '');
...
}

static createDetailLine(ul, labelText, value) {
  ...
}
```